### PR TITLE
Imx6 usb

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

File: src/drivers/usb/hc/ehci_regs.h [hunk 1]
```diff
@@ -28,6 +31,8 @@
 #define EHCI_HCC_PGM_FRAMELISTLEN(p)      ((p) & (1 << 1))   /* true: periodic_size changes*/
 #define EHCI_HCC_64BIT_ADDR(p)            ((p) & (1))        /* true: can use 64-bit addr */
 
+
```
Comment (Deryugin) [inline src/drivers/usb/hc/ehci_regs.h]: Triple blank lines

File: src/drivers/usb/hc/usb_ehci_mxc_regs.h [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ * @file
+ * @brief Registers definition
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version
+ * @date 26.06.2019
+ */
+
+#ifndef IMX_USB_REGS_H_
+#define IMX_USB_REGS_H_
+
```
Comment (Deryugin) [inline src/drivers/usb/hc/usb_ehci_mxc_regs.h]: Double blank line

File: src/drivers/usb/hc/usb_ehci_mxc_regs.h [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# define USB_UOG_ID_NID_MASK 0x3f
+# define USB_UOG_ID_ID_OFFT  0
+# define USB_UOG_ID_ID_MASK  0x3f
+
+#define USB_UOG_HWGENERAL    (0x004)
+#define USB_UOG_HWHOST       (0x008)
+#define USB_UOG_HWDEVICE     (0x00C)
+#define USB_UOG_HWTXBUF      (0x010)
+#define USB_UOG_HWRXBUF      (0x014)
+
```
Comment (Deryugin) [inline src/drivers/usb/hc/usb_ehci_mxc_regs.h]: Double blank line

File: src/drivers/usb/hc/ehci_regs.h (hunk 1 above)
Comment (anton-bondarev) [reply src/drivers/usb/hc/ehci_regs.h]: done

File: src/drivers/usb/hc/usb_ehci_mxc_regs.h (hunk 2 above)
Comment (anton-bondarev) [reply src/drivers/usb/hc/usb_ehci_mxc_regs.h]: done

File: src/drivers/usb/hc/usb_ehci_mxc_regs.h (hunk 3 above)
Comment (anton-bondarev) [reply src/drivers/usb/hc/usb_ehci_mxc_regs.h]: done

File: src/drivers/usb/hc/ehci_mxc.c [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+		imx_usb_phy_enable(i);
+		imx_usb_powerup(i);
+
+		ehci_mxc_init(&ehci_mxc_hcs[i]);
+
+		ehci_hcd_register((void *) (ehci_mxc_hcs[i].base + 0x100), ehci_mxc_hcs[i].irq_num);
+	}
+
+	log_boot_stop();
+
```
Comment (Deryugin) [inline src/drivers/usb/hc/ehci_mxc.c]: Double blank line

Comment (anton-bondarev) [reply src/drivers/usb/hc/ehci_mxc.c]: done

File: src/drivers/usb/hc/ehci_mxc.c [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+#define IMX6_USB_OTG_IRQ    75
+#define IMX6_USB_PHY_UTMI0  76
+#define IMX6_USB_PHY_UTMI1  77
+
+
+static inline void imx_usb_phy_enable(int port) {
+	int tmp;
+
+	REG32_STORE(USBPHY_CTRL_SET(port), USBPHY_CTRL_SFTRST);
+	printk("trace %s:%d\n", __func__, __LINE__);
```
Comment (Deryugin) [inline src/drivers/usb/hc/ehci_mxc.c]: Remove this debug output, please

**Pull request merged**